mep/people/forms.py

```diff
@@ -100,6 +108,23 @@ class MemberSearchForm(FacetForm):
     sex = FacetChoiceField(label='Gender', widget=CheckboxFieldset(attrs={
         'class': 'choice facet'
     }))
+    membership_dates = RangeField(label='Membership Dates', required=False,
+        widget=RangeWidget(attrs={'size': 4}))
+
+    def set_membership_dates_placeholder(self, min_year, max_year):
+        '''Set the min, max, and placeholder values for
+        :class:`mep.common.forms.RangeWidget` associated with membership_dates.'''
+
+        min_widget, max_widget = self.fields['membership_dates'].widget.widgets
```

Comment:
This is pretty minor, but I think start/end would be clearer for the widgets, since that's how we think of them in the field and the widget.

Reply:
As in just `start` `end` simply or `start_widget`, etc.?

Reply:
Not just start/end; `start_widget` or `start_input` or something like that?